microsoft/nni
# Refactor kill job

### Description ###
- use fluent model instead of browser alert model
- function component use `context` directly
- update `website update`

### Progress ###
  - [x] disable refresh and kill job https://github.com/microsoft/nni/pull/4772#discussion_r853739007 

### After comments ###
- Delete `copyParameter.scss` file because it is for ant-design pro
- Put `color: #333` in common.scss file because had too many `color: #333` in webui

### How to test ###
Run example, 
- kill trial -> observe table status update;
- close the page auto refresh or change interval value -> kill trial -> observe table status update;
   
![image](https://user-images.githubusercontent.com/61399850/164365489-3d12c09b-310d-4d80-91f4-2c2b3ec86fb3.png)

- test kill job by your website usage habits
